.gitmodules

```diff
@@ -77,3 +77,6 @@
 	path = Sming/Libraries/DHTesp
 	url = https://github.com/beegee-tokyo/DHTesp.git
 	ignore = dirty
+[submodule "Sming/third-party/ITEADLIB_Arduino_Nextion"]
+	path = Sming/Libraries/ITEADLIB_Arduino_Nextion
+	url = https://github.com/itead/ITEADLIB_Arduino_Nextion.git
```

Comment:
@vijaygill Add one more line here with content:
ignore = dirty

similar to the other repositories.